_layouts/meetup.html

```diff
@@ -9,10 +9,14 @@
 
     <article class="meetup">
         <header class="meetup__header">
-            <h1>{{ page.title }}</h1>
-            <h2>{{ page.description }}</h2>
-			<h3>{{ page.speaker.name }}</h3>
-            <h4>{{ page.date | date:"%B %-d, %Y" }} | {{ page.attendees }} asistentes | {{ page.meetupinfo.location }} </h4>
+            <h1 class="meetup__title">{{ page.title }}</h1>
+            <h2 class="meetup__description">> {{ page.description }}</h2>
+            <h3 class="meetup__details">
+                <span class="meetup__date">{{ page.date | date:"%B %-d, %Y" }}</span>
+                <span class="meetup__attendees">{{ page.attendees }} asistentes</span>
+                <span class="meetup__location">{{ page.meetupinfo.location }}</span>
+            </h3>
+            <h4 class="meetup__speaker">> {{ page.speaker.name }}</h4>
```

Comment:
Sobra un cierre de tag

Reply:
the same

Reply:
mira las capturas de meetup y post para que  veas porque existe ese >